actors/builtin/miner/miner_actor.go

```diff
@@ -1417,18 +1441,27 @@ func (a Actor) WithdrawBalance(rt Runtime, params *WithdrawBalanceParams) *adt.E
 		if err != nil {
 			rt.Abortf(exitcode.ErrIllegalState, "failed to vest fund: %v", err)
 		}
+		unlockedBalance = st.GetUnlockedBalance(rt.CurrentBalance())
 
-		// Verify InitialPledgeRequirement does not exceed unlocked funds
-		verifyPledgeMeetsInitialRequirements(rt, &st)
+		// Verify unlocked funds cover both InitialPledgeRequirement and FeeDebt
+		// and repay fee debt now.
+		feeDebtPenalty = verifyPledgeRequirementsAndRepayDebts(rt, &st)
+		unlockedBalance = big.Sub(unlockedBalance, feeDebtPenalty)
```

Comment:
Similar comment to above: repay fees, then calculate available balance.

And again, unlocked rather than available here seems wrong.